index.html

```diff
@@ -3686,46 +3692,42 @@ <h1>Resolution</h1>
-URL dereferencing</a>. These functions are defined in an abstract way. Their
-exact implementation is out of scope for this specification, but some
-considerations for implementors are discussed in [[?DID-RESOLUTION]].
+URL dereferencing</a>.  Their exact implementation is out of scope for this
+specification, but some considerations for implementors are discussed in
+[[?DID-RESOLUTION]].
     </p>
 
     <p>
 All conformant <a>DID resolvers</a> MUST implement the <a>DID resolution</a>
-functions for at least one <a>DID method</a> and MUST be able to return a <a>DID
-document</a> in at least one conformant <a>representation</a>.
+functions for at least one <a>DID method</a> and MUST be able to produce a
+<a>DID document</a> in at least one conformant <a>representation</a>.
     </p>
 
-    <section>
+    <section data-dfn-for="Resolver">
         <h2>DID Resolution</h2>
         <p>
-The <a>DID resolution</a> functions resolve a <a>DID</a> into a <a>DID
-document</a> by using the "Read" operation of the applicable <a>DID method</a>.
-(See <a href="#read-verify"></a>.) The details of how this process is
-accomplished are outside the scope of this specification, but all conformant
-implementations implement two functions which have the following abstract
-forms:
+The <a>DID resolution</a> function resolves a <a>DID</a> into a <a>DID
+document</a> by using the "Read" operation of the applicable <a>DID method</a>
+as described in <a href="#read-verify"></a>. The details of how this process is
+accomplished are outside the scope of this specification, but all conforming
+<a>DID resolvers</a> implement the functions below which have the following
+abstract forms:
         </p>
 
-        <p><code>
-resolve ( did, did-resolution-input-metadata ) <br>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&gt; ( did-resolution-metadata, did-document, did-document-metadata )
-        </code></p>
+        <pre title="Abstract functions for DID Resolution">
+resolve(did, options) →
+   « didResolutionMetadata, didDocument, didDocumentMetadata »
 
-        <p><code>
-resolveRepresentation ( did, did-resolution-input-metadata ) <br>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&gt; ( did-resolution-metadata, did-document-stream, did-document-metadata )
-        </code></p>
+resolveRepresentation(did, options) →
+   « didResolutionMetadata, didDocumentStream, didDocumentMetadata »
+        </pre>
 
         <p>
 The <code>resolve</code> function returns the <a>DID document</a> in its
-abstract form.
-The <code>resolveRepresentation</code> function returns a byte stream of the
-<a>DID Document</a> formatted in the corresponding representation.
-        </p>
-        <p>
-The input variables of these functions are as follows:
+abstract form. The <code>resolveRepresentation</code> function returns a byte
+stream of the <a>DID Document</a> formatted in the corresponding representation.
+The input variables of the <code>resolve</code> and
+<code>resolveRepresentation</code> functions are as follows:
```

Comment:
```suggestion
<code>resolveRepresentation</code> functions are as follow:
```
"{singular} is as follows" or "{plural} are as follow"

Reply:
This is not correct, the correct form is "as follows" regardless of the number of elements.

Reply:
Huh... The gurus of the web concur with you, and much younger me. I wish I'd had Google when in school...